src/mon/Monitor.cc

```diff
@@ -5584,14 +5583,19 @@ int Monitor::scrub_start()
   dout(10) << __func__ << dendl;
   ceph_assert(is_leader());
 
-  if (!scrub_result.empty()) {
-    clog->info() << "scrub already in progress";
-    return -EBUSY;
+  if (scrub_con.load() != nullptr) {
```

Comment:
Do we need the double check? How about:

```suggestion
 if (std::shared_ptr<ScrubContext> local_ctx = scrub_con.load(); scrub_ctx.load() != nullptr) {
```

Reply:
done